receipt_langsmith/receipt_langsmith/queries.py

```
@@ -0,0 +1,230 @@
+        hours_back: How many hours back to search
+
+    Returns:
+        List of receipt info dicts with all evaluation decisions
+    """
+    client = get_langsmith_client()
+    traces = query_recent_receipt_traces(project_name, hours_back)
+
+    interesting_receipts = []
+
+    for trace in traces:
+        outputs = trace.outputs or {}
```

Ruff (0.14.10)

Local variable `outputs` is assigned to but never used

Remove assignment to unused variable `outputs`

(F841)